hack/make/.go-autogen

```diff
@@ -8,11 +8,12 @@ cat > autogen/dockerversion/dockerversion.go <<EOF
 package dockerversion
 
 var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
+	GITCOMMIT    string = "$GITCOMMIT"
+	VERSION      string = "$VERSION"
 
-	IAMSTATIC string = "${IAMSTATIC:-true}"
-	INITSHA1  string = "$DOCKER_INITSHA1"
-	INITPATH  string = "$DOCKER_INITPATH"
+	IAMSTATIC    string = "${IAMSTATIC:-true}"
+	INITSHA1     string = "$DOCKER_INITSHA1"
+	INITPATH     string = "$DOCKER_INITPATH"
+	EXPERIMENTAL   bool = $EXPERIMENTAL
```

Comment:
yeahhh i think I was half asleep when I wrote this, I do like this api better but theres a better way to do this